Dockerfile

```diff
@@ -12,12 +12,12 @@ RUN INSTALL_PKGS=" \
     yum clean all && \
     mkdir /tmp/ocp-clo && \
     chmod og+w /tmp/ocp-clo
-COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/_output/bin/cluster-logging-operator /usr/bin/
+COPY cmd/manager /usr/bin/cluster-logging-operator
```

Comment:
this defeats the whole purpose of using a builder image

Reply:
i think this would break ART -- they do not have access to our `Makefile`

Reply:
This will break the ART process because this copies the binary from local instead of the building layer.

Reply:
OK, will fix. My bad.